web/l10n/en.json

```diff
@@ -205,5 +207,7 @@
     "categoryPlane": "Plane",
     "categoryMotorcycle": "Motorcycle",
     "categoryBicycle": "Bicycle",
-    "categoryPerson": "Person"
+    "categoryPerson": "Person",
+    "calendarFile": "Calendar File",
```

Comment:
I'm thinking that maybe we should make these two string more common. Something like "File" and "Select File". It would reduce number of string if we decide to use it for something else in future.